src/main.js

```diff
@@ -0,0 +1,502 @@
+            </div>
+
+            <div class="card__colors-inner">
+              <h3 class="card__colors-title">Color</h3>
+              <div class="card__colors-wrap">
+                <input
+                  type="radio"
+                  id="color-black-4"
+                  class="card__color-input card__color-input--black visually-hidden"
+                  name="color"
+                  value="black"
+                />
+                <label
+                  for="color-black-4"
+                  class="card__color card__color--black"
+                  >black</label
+                >
+                <input
+                  type="radio"
+                  id="color-yellow-4"
+                  class="card__color-input card__color-input--yellow visually-hidden"
+                  name="color"
+                  value="yellow"
+                  checked
+                />
+                <label
+                  for="color-yellow-4"
+                  class="card__color card__color--yellow"
+                  >yellow</label
+                >
+                <input
+                  type="radio"
+                  id="color-blue-4"
+                  class="card__color-input card__color-input--blue visually-hidden"
+                  name="color"
+                  value="blue"
+                />
+                <label
+                  for="color-blue-4"
+                  class="card__color card__color--blue"
+                  >blue</label
+                >
+                <input
+                  type="radio"
+                  id="color-green-4"
+                  class="card__color-input card__color-input--green visually-hidden"
+                  name="color"
+                  value="green"
+                />
+                <label
+                  for="color-green-4"
+                  class="card__color card__color--green"
+                  >green</label
+                >
+                <input
+                  type="radio"
+                  id="color-pink-4"
+                  class="card__color-input card__color-input--pink visually-hidden"
+                  name="color"
+                  value="pink"
+                />
+                <label
+                  for="color-pink-4"
+                  class="card__color card__color--pink"
+                  >pink</label
+                >
+              </div>
+            </div>
+          </div>
+
+          <div class="card__status-btns">
+            <button class="card__save" type="submit">save</button>
+            <button class="card__delete" type="button">delete</button>
+          </div>
+        </div>
+      </form>
+    </article>
+  `);
+};
+
+// кнопка LoadMore
+const createLoadMoreTemplate = () => {
+  return (`
+    <button class="load-more" type="button">load more</button>
+  `);
+};
+
+
+//  ---------------------- ф-ция рендер --------------------------------
+const render = (container, template, place) => {
```

Comment:
```suggestion
const render = (container, template, place = `beforeend`) => {
```
Заметь, как часто ты вызываешь эту функцию с параметром `beforeend`.
Было бы круто, поставить значение по умолчанию и добавлять при вызове 3й параметр, только когда он другой.